SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -1618,3 +1611,50 @@ void USpatialReceiver::ResolveObjectReferences(FRepLayout& RepLayout, UObject* R
 		}
 	}
 }
+
+void USpatialReceiver::OnHeartbeatComponentUpdate(Worker_ComponentUpdateOp& Op)
+{
+	if (!NetDriver->IsServer())
+	{
+		// Clients can ignore Heartbeat component updates.
+		return;
```

Comment:
Not even sure this would ever get hit because clients can't see other PlayerControllers but it's probably fine to keep this.